modules/nextflow/src/main/groovy/nextflow/util/LoggerHelper.groovy

```diff
@@ -437,7 +437,7 @@ class LoggerHelper {
     static protected void appendFormattedMessage( StringBuilder buffer, ILoggingEvent event, Throwable fail, Session session) {
         final className = session?.script?.getClass()?.getName()
         final message = event.getFormattedMessage()
-        final quiet = fail instanceof AbortOperationException || fail instanceof ProcessException || ScriptRuntimeException
+        final quiet = fail instanceof AbortOperationException || fail instanceof ProcessException || fail instanceof ScriptRuntimeException
```

Comment:
@pditommaso still should commit this change manually, it's a general bug fix

Reply:
I see it now, thanks for spotting it. I'm going to apply this patch